rootfs/etc/nginx/lua/monitor.lua

```diff
@@ -1,12 +1,15 @@
 local socket = ngx.socket.tcp
 local cjson = require('cjson')
 local assert = assert
+local new_tab = require "table.new"
+local clear_tab = require "table.clear"
 
-local metrics_batch = {}
 -- if an Nginx worker processes more than (MAX_BATCH_SIZE/FLUSH_INTERVAL) RPS then it will start dropping metrics
 local MAX_BATCH_SIZE = 10000
 local FLUSH_INTERVAL = 1 -- second
 
+local metrics_batch = new_tab(MAX_BATCH_SIZE, 0)
```

Comment:
What's the second argument for? Why 0? A link to docs would be sufficient too.

Reply:
doc: https://github.com/openresty/luajit2/blob/v2.1-agentzh/doc/extensions.html
`table.new(narray, nhash)`, the second argument is for hash.